sklearn/utils/tests/test_validation.py

```diff
@@ -2124,3 +2125,13 @@ def __init__(self):
             self.schema = ["a", "b"]
 
     assert not _is_polars_df(LooksLikePolars())
+
+
+def test_pydata_sparse_dispatch():
```

Comment:
Note that the name is a bit misleading. It's not dispatching anything to pydata/sparse but rather convert to scipy sparse.